src/app/app.component.ts

```diff
@@ -1,6 +1,6 @@
 import { CUSTOM_ELEMENTS_SCHEMA, Component, OnDestroy, OnInit } from '@angular/core';
 import { FooterComponent } from './components/footer/footer.component';
-import { Router, RouterOutlet,RouterLink } from '@angular/router';
+import { Router, RouterOutlet,RouterLink, ActivatedRoute } from '@angular/router';
```

Comment:
I can't see that this ActivatedRoute is used anywhere?

Reply:
Your right, I changed that and forgot to remove it. Good catch!